documentation/tutorials/get-started-with-phoenix.md

```diff
@@ -0,0 +1,467 @@
+    ]
+  end
+```
+
+Each Ash package also has its own formatter extension, include these in your `.formatter.exs` to ensure consistent formatting.
+
+```elixir
+# .formatter.exs
+[
+  import_deps: [:ecto, :phoenix, :ash, :ash_phoenix, :ash_postgres],
+  inputs: ["*.{ex,exs}", "priv/*/seeds.exs", "{config,lib,test}/**/*.{ex,exs}"],
+  subdirectories: ["priv/*/migrations"]
+]
+```
+
+Now in the terminal install these new dependencies:
+
+```bash
+mix deps.get
+```
+
+### If using Phoenix 1.7 or higher
+
+we need to change `helpers: false` to `helpers: true` in the router section:
+
+```elixir
+# lib/example_web.ex
+
+defmodule PhoenixHelpdeskWeb do
+# ...
+  def router do
+    quote do
+      use Phoenix.Router, helpers: true # <-- Change this line
+    # ...
+    end
+  end
+end
+```
+
+### Swap `Ecto.Repo` for `AshPostgres.Repo`
+
+Change your repo module to look like this:
+
+```elixir
+# lib/phoenix_helpdesk/repo.ex
+
+defmodule PhoenixHelpdesk.Repo do
+  use AshPostgres.Repo, opt_app: :phoenix_helpdesk
+
+  def installed_extensions do
+    ["uuid-ossp", "citext"]
+  end
+end
+```
+
+This now means Phoenix connects to AshPostgres instead of Ecto.
+
+### Edit Config
+
+We need to specify the apis that ash uses and point to the Ecto repo we just changes into an AshPostgres repo.
+
+```elixir
+import Config
+
+# For backwards compatibility, the following configuration is required.
+# see https://ash-hq.org/docs/guides/ash/latest/get-started#temporary-config for more details
+config :ash, :use_all_identities_in_manage_relationship?, false
+
+config :phoenix_helpdesk,
+  ash_apis: [PhoenixHelpdesk.Support]
+
+config :phoenix_helpdesk,
+  ecto_repos: [PhoenixHelpdesk.Repo]
+```
+
+### Create API
+
+You may have noticed that `PhoenixHelpdesk.Support` doesn't actually exist yet. Lets make it now.
+
+```elixir
+# lib/phoenix_helpdesk/support.ex
+
+defmodule PhoenixHelpdesk.Support do
+  use Ash.Api
+
+  resources do
+    registry PhoenixHelpdesk.Support.Registry
+  end
+end
+```
+
+This tiny module tells Ash that`PhoenixHelpdesk.Support` is an Ash API. It can be thought of as a [Bounded Context](https://martinfowler.com/bliki/BoundedContext.html) in Domain Driven Design terms and can seen as analogous to a Phoenix context.
+
+It also tells where to find the resources within this API - in the `PhoenixHelpdesk.Support.Registry` which we will make now.
+
+### Create Registry
+
+```elixir
+# lib/phoenix_helpdesk/support/registry.ex
+
+defmodule PhoenixHelpdesk.Support.Registry do
+  use Ash.Registry,
+    extensions: [
+      # This extension adds helpful compile time validations
+      Ash.Registry.ResourceValidations
+    ]
+
+  entries do
+    entry PhoenixHelpdesk.Support.Ticket
+    entry PhoenixHelpdesk.Support.Representative
+  end
+end
+```
+
+---
+
+NOTE TO SELF
+
+Should I stop this first part here? Ash is now fully installed it's just not doing anything. It would be a good place to stop?
```

Comment:
I think just keep going :D We already have the individual getting started guides that do simple getting started stuff in a very "piecemeal" way.

Reply:
Ahah yea I've been trying to think about how to structure this, I just want to make it as small as possible. And then build on it with more guides. Its more consumable that way.